database/youtube/web-development.json

```diff
@@ -61,5 +61,22 @@
         "category": "youtube",
         "subcategory": "web-development",
         "language": "hindi"
+    },
+    {
+        "name": "Bro Code",
+        "description": "This channel is provide a web development course for both beginner and pros.",
```

Comment:
Hi @darkweb907! 😄 Congrats on starting your first open source contribution! 🥳 This is a great start. However, I highly recommend explaining what makes this channel stand out from other Youtube channels that teach web development. This will pique the interest of users.  
Happy coding! 😄